<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Vaibhav-HP" failures="1" tests="1" name="edgeTest" time="12.881" errors="0" timestamp="2024-09-08T00:18:43 IST">
  <testcase classname="testCases.TC_LoginTest" name="verify_login" time="1.791">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testCases.TC_LoginTest.verify_login(TC_LoginTest.java:39)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
... Removed 17 stack frames]]>
    </failure>
  </testcase> <!-- verify_login -->
</testsuite> <!-- edgeTest -->
